### PR TITLE
Search for projects and organizations

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

**Review by gigxz — APPROVED** [review]

File: drivers/hmis/app/models/hmis/hud/client.rb [hunk 1]
```diff
@@ -82,8 +82,8 @@ def self.client_search(input:, user: nil)
     scope = scope.where(c_t[:SSN].matches("%#{input.ssn_serial}")) if input.ssn_serial.present?
     scope = scope.where(c_t[:DOB].eq(Date.parse(input.dob))) if input.dob.present?
 
-    # TODO: projects
-    # TODO: organizations
+    scope = scope.joins(:projects).merge(Hmis::Hud::Project.viewable_by(user).where(id: input.projects)) if input.projects.present?
+    scope = scope.joins(projects: :organization).merge(Hmis::Hud::Organization.viewable_by(user).where(id: input.organizations)) if input.organizations.present?
```
Comment (gigxz) [inline drivers/hmis/app/models/hmis/hud/client.rb:86]: Sorry @eanders I didn't actually test it manually when reviewing, I assumed you had. This throws an error, I think it needs to join with enrollments first?
> ActiveRecord::ConfigurationError (Can't join 'GrdaWarehouse::Hud::Client' to association named 'projects'; perhaps you misspelled it?)

Comment (eanders) [reply drivers/hmis/app/models/hmis/hud/client.rb:86]: Odd, I did test, it worked fine, didn't throw any issues.
`has_many :projects, through: :enrollments` is on line 48

Comment (gigxz) [reply drivers/hmis/app/models/hmis/hud/client.rb:86]: It's searching `GrdaWarehouse::Hud::Client` at this point (which doesn't appear to have that assocation) and then getting `Hmis::Hud::Client`s on line 88

Comment (eanders) [reply drivers/hmis/app/models/hmis/hud/client.rb:86]: Ok, I must have gotten really lucky somehow.  I'll update.

**Pull request merged**